src/components/TradeWidget/OrderValidity.tsx

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
This is a really big file. Consider moving some stuff outside, like styled-components and what not.

Reply:
definitely, may need to tackle that after this PR as it's pending so long